message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -20,6 +20,7 @@
    [cmr.message-queue.config :as config]
    [cmr.message-queue.services.queue :as queue])
   (:import
+   (cmr.message_queue.test ExitException)
```

Comment:
That is a src namespace. src/cmr/message_queue/test holds three namespaces that support tests, including cmr.message-queue.test.ExitException.